### DISCUSSION TITLE
Skin Searchbar

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#14332] [Ideas]
If you've downloaded like 500 skins, and you want to search for just one, it can get tedious having to scroll through so many. My proposal is for there to be a searchbar (prob above the skins dropdown menu) that allows us to search for a skin.

**Comment by Joehuu** [comment]
~~Can't you already do that via the settings searchbar?~~

Edit: actually it doesn't auto scroll.

<img width="453" alt="Screen Shot 2021-08-15 at 9 38 52 PM" src="https://user-images.githubusercontent.com/35318437/129512135-193f9096-df0e-4524-819b-287de3a2410d.png">

**Comment by peppy** [comment]
We do have plans for a much better skin selection interface, although I don't think we have anything from @arflyte yet.

**Comment by arflyte** [comment]
In the control panel itself? We have draft design for skin panels but not in the control panel itself. https://r.flyte.moe/2108/Figma_210816-164327.png

**Reply by peppy** [reply]
Basically yeah. Either inline in the panel, or in its own subpanel (maybe a better option) similar to key bindings. I'd expect it to give users a preview of skins rather than just a dropdown.
